.pyproject_generation/pyproject_custom.toml

```diff
@@ -1,12 +1,12 @@
 [project]
 name = "mass"
-version = "2.0.0"
+version = "2.0.1"
 description = "Metadata Artifact Search Service  - A service for searching metadata artifacts and filtering results."
 dependencies = [
     "typer>=0.9.0",
-    "ghga-service-commons[api]>=2.0.0",
-    "ghga-event-schemas>=1.0.0",
-    "hexkit[mongodb,akafka]>=2.0.0",
+    "ghga-service-commons[api]>=3.0.0",
+    "ghga-event-schemas>=2.0.0",
```

Comment:
Any reason to not move to v3?

Reply:
Because it works with v2. So v3 isn't *required*, and the update lock process will use the latest available version anyway. I was trying to follow the SOP, but admittedly only the version in my memory so it may be wrong.